### DISCUSSION TITLE
Thanks for your hard work on this project.

### DISCUSSION THREAD

**Discussion by GlacierFox** [microsoft/fluentui-blazor#2126] [Show and tell]
Hey, just popping in to show a little appreciation for the time and effort you guys are putting on this. Really enjoyed the recent Community Standup with @vnbaaij and @dvoituron  [here](https://www.youtube.com/watch?v=1fveBAi6Q7I&) too. 

It's nothing to shout about but I've built my own internal site using `fluentui-blazor` for me and my friends to use which links to an ASP.NET API for development purposes and it's turned out pretty nice.

![image](https://github.com/microsoft/fluentui-blazor/assets/9074086/ca832e17-9329-48ad-9b11-8b3a8f459d99)

Looking forward to the Fluent 2 stuff mentioned in the video. 👏

**Comment by vnbaaij** [comment]
Looking good!

**Comment by dvoituron** [comment]
Awesome. Congrats.
